core/src/main/res/values/styles.xml

```diff
@@ -1,11 +1,17 @@
 <?xml version="1.0" encoding="utf-8"?>
-<resources xmlns:tools="http://schemas.android.com/tools" xmlns:android="http://schemas.android.com/apk/res/android">
+<resources xmlns:android="http://schemas.android.com/apk/res/android" xmlns:tools="http://schemas.android.com/tools">
 
     <style name="Theme.AntennaPod.Light" parent="Theme.Base.AntennaPod.Light">
         <!-- Room for API dependent attributes -->
     </style>
 
     <style name="Theme.Base.AntennaPod.Light" parent="Theme.MaterialComponents.Light">
+        <item name="filter_dialog_button_background">
+            @drawable/filter_dialog_button_background_light
+        </item>
+        <item name="filter_dialog_button_clear">@drawable/filter_dialog_button_clear</item>
+        <item name="filter_dialog_clear_active">@color/dialog_filter_clear_active_light</item>
+        <item name="filter_dialog_clear_inactive">@color/dialog_filter_clear_inactive_light</item>
```

Comment:
I would prefer if you could move this to the bottom of the theme definition. I think that the primary accent color should be on top, so that it is easier to see the key aspects of the theme directly.

Reply:
:+1: